CHANGELOG.md

```diff
@@ -14,8 +14,20 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
-## [29.74.4] - 2025-09-03
-- Add a pre-check step for initializing xDS stream connections
+## [29.75.3] - 2025-09-05
```

Comment:
this needs minor version bump since container needs to pass config to it, to do general check for all standard d2 clients too.